### DISCUSSION TITLE
Public functions from `InteractsWithMedia` are missing in `HasMedia`

### DISCUSSION THREAD

**Discussion by korridor** [spatie/laravel-medialibrary#3426] [Ideas]
Functions like `addMediaFromStream` or `addMediaFromDisk` are missing in the interface `HasMedia`. This produces a lot of warnings when using PhpStan, since we are using the interface for example in function definitions.

<img width="816" alt="image" src="https://github.com/spatie/laravel-medialibrary/assets/26689068/4e9592a3-8c89-4a0a-834d-17a156696527">

Would you be interested in an PR for that?

**Comment by freekmurze** [comment]
Yeah, sure send a PR.
